### PR TITLE
Html5 tag aria

### PR DESCRIPTION
**Pull request by Lillian-wang**

### CODE REVIEW

File: app/views/articles/_article_item.html.erb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-    <h3><%= link_to(article.title, article.url, target: "_blank") %></h3>
-    <%= link_to("Show Page", article) %>
-    <%= image_tag article.image_url(:thumb).to_s %> 
-  </div>
-</li>
\ No newline at end of file
+<li class="article-item" >
+  <article>
+    <a href="<%= article.url %>">
+      <figure aira-hidden = "true">
```
Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: spelling mistake

Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: figure tag is unnecessary here. Please remove

File: app/views/articles/_article_item.html.erb [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+        <%= image_tag article.image, {:class =>"article-img",:role => "button"} %>
```
Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: remove role=button. This is primarily an image.

File: app/views/articles/_article_item.html.erb [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+      </figure>
+    </a>
+    <div class="article-info">
+      <header>
```
Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: Remove header

File: app/views/articles/_article_item.html.erb [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+        <h3 ><%= link_to(article.title, article.url, target: "_blank") %></h3>
+        <%= link_to("Show Page", article) %>
+      </header>
+      <figure aira-hidden = "true">
```
Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: remove figure tag

File: app/views/articles/_article_item.html.erb (hunk 2 above)
Comment (comolongo) [inline app/views/articles/_article_item.html.erb]: This image should be a thumb

File: app/views/articles/_comment.html.erb [hunk 5]
```diff
@@ -1,10 +1,10 @@
 <li class="comments-stack"> 
     <span class="author-name"><%= comment.author.username %></span>
-    <span class="submit-time"> Submitted <%= time_ago_in_words(comment.created_at) %> ago</span>
+    <time class="submit-time"> Submitted <%= time_ago_in_words(comment.created_at) %> ago</time>
```
Comment (comolongo) [inline app/views/articles/_comment.html.erb]: Add datetime attribute

File: app/views/articles/index.html.erb [hunk 6]
```diff
@@ -1,46 +1,11 @@
...
-</div>
\ No newline at end of file
+<main>  
+    <h1>Articles</h1>
+    <ul class="articles-content">
+    <% @articles.each do |article| %>
+      <%= render partial: "article_item", collection: @articles, as: :article %>
+    <% end %>
+    </ul>
+  <div id="load-more">
```
Comment (comolongo) [inline app/views/articles/index.html.erb]: indent

File: app/views/articles/new.html.erb [hunk 7]
```diff
@@ -1,15 +1,15 @@
-<div id="new-article-main">
+<main id="new-article-main">
   <%= form_for @article, url: articles_path(@article), :html => {:multipart => true} do |article|  %>
-    <div class="wrapper">
-      <label class="input-section">
+    <div class="wrapper" role="listbox" >
```
Comment (comolongo) [inline app/views/articles/new.html.erb]: no need for role=listbox

File: app/views/articles/new.html.erb [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
+      <label class="input-section" role="option">
```
Comment (comolongo) [inline app/views/articles/new.html.erb]: remove options as well

File: app/views/articles/_article_item.html.erb (hunk 3 above)
Comment (Lillian-wang) [inline app/views/articles/_article_item.html.erb]: done

File: app/views/articles/_article_item.html.erb (hunk 2 above)
Comment (Lillian-wang) [inline app/views/articles/_article_item.html.erb]: done

File: app/views/articles/_article_item.html.erb (hunk 1 above)
Comment (Lillian-wang) [inline app/views/articles/_article_item.html.erb]: done

File: app/views/articles/_article_item.html.erb (hunk 4 above)
Comment (Lillian-wang) [inline app/views/articles/_article_item.html.erb]: done

File: app/views/articles/_comment.html.erb (hunk 5 above)
Comment (Lillian-wang) [inline app/views/articles/_comment.html.erb]: done

File: app/views/articles/index.html.erb (hunk 6 above)
Comment (Lillian-wang) [inline app/views/articles/index.html.erb]: done

**Pull request closed without merging**